### PR TITLE
dev-tex/culmus-latex: EAPI7 revbump, improve ebuild

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This Bug/PR updates culmus-latex for EAPI7.
Please review

Closes: https://bugs.gentoo.org/664520

### CODE REVIEW

File: dev-tex/culmus-latex/culmus-latex-0.7-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+RDEPEND="virtual/latex-base"
+DEPEND="${RDEPEND}
+	|| (
+		>=media-fonts/culmus-0.110[fancy]
+		<media-fonts/culmus-0.110
+	)
+	app-text/t1utils"
+
+DOCS=( README )
```
Comment (Zlogene) [inline dev-tex/culmus-latex/culmus-latex-0.7-r1.ebuild]: dodoc

File: dev-tex/culmus-latex/culmus-latex-0.7-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+src_install() {
+	emake CULMUSDIR=/usr/share/fonts/culmus/ \
+		DESTDIR="${D}" \
+		TEXMFROOT=/usr/share/texmf-site \
+		pkginstall
+
+	insinto /etc/texmf/updmap.d
+	doins ${PN}.cfg
+
+	einstalldocs
```
Comment (Zlogene) [inline dev-tex/culmus-latex/culmus-latex-0.7-r1.ebuild]: not needed

File: dev-tex/culmus-latex/culmus-latex-0.7-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit latex-package
+
+DESCRIPTION="Culmus fonts support for latex"
+HOMEPAGE="http://ivritex.sourceforge.net/"
+SRC_URI="mirror://sourceforge/ivritex/${P}_src.tar.gz"
```
Comment (Zlogene) [inline dev-tex/culmus-latex/culmus-latex-0.7-r1.ebuild]: `-> ${P}.tar.gz`

**Comment by mm1ke** [comment]
ping, any chance we can merge this :)

**Pull request closed without merging**